components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/java/org/wso2/carbon/identity/api/server/script/library/v1/core/ServerScriptLibrariesService.java

```diff
@@ -280,12 +281,19 @@ public boolean isScriptLibraryAvailable(String scriptLibraryName) {
      */
     private ScriptLibraryResponse createScriptLibraryResponse(FunctionLibrary scriptLibrary) {
 
+        String displayName;
```

Comment:
This variable initialization is not needed.

Reply:
Fixed in f72b9a5